_posts/2020-08-20-snowplow-javascript-tracker-2.15.0-released/index.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+title: "Snowplow JavaScript Tracker 2.15.0 released"
+title-short: Snowplow JavaScript Tracker 2.15.0
+tags: [snowplow, javascript, tracker]
+author: Paul
+category: Releases
+permalink: /blog/2020/08/20/snowplow-javascript-tracker-2.15.0-released/
+discourse: false
+---
+
+We are pleased to announce a new release of the [Snowplow JavaScript Tracker][js-tracker].
+
+[Version 2.15.0][2.15.0-tag] introduces two key features around user identification: anonymous tracking capabilities and support for Chrome's Client Hints (that will replace the traditional useragent string). It also comes with some minor improvements and bug fixes.
+
+Read on below for:
+
+1. [Anonymous Tracking](#1-anonymous-tracking)
+2. [Support for Chrome's Client Hints](#2-client-hints)
+3. [Other features and bug fixes](#3-other-features-and-bugfixes)
+4. [Upgrading](#4-upgrading)
+5. [Documentation and help](#5-documentation-and-help)
+
+<!--more-->
+
+## 1. Anonymous Tracking
+
+The Snowplow technology is built from the ground up for maximum flexibility and control. In an age of incerasing awareness of user privacy, this should include giving users the ability to decide whether or not they want to track persistent user identifiers with their Snowplow web events. Therefore, we are introducing anonymous tracking capabilities into the Snowplow JavaScript tracker ([Github issue](https://github.com/snowplow/snowplow-javascript-tracker/issues/793)). Specifically, we are providing two techniques, full anonymous tracking and anonymous session tracking. By default `anonymousTracking: false`.
+
+Recommended configurations when using anonymousTracking:
+
+```
+anonymousTracking: true,
+stateStorageStrategy: 'cookieAndLocalStorage'
+```
+
+or
+
+```
+anonymousTracking: { withSessionTracking: true },
+stateStorageStrategy: 'cookieAndLocalStorage'
+```
+
+If using the anonymous session tracking, the session key in local storage will also contain a salt which is used for stitching together the sessions. This value is never sent to the collector.
+
+More information on how to instrument anonymous tracking can be found in the [technical documentation](https://docs.snowplowanalytics.com/docs/collecting-data/collecting-from-own-applications/javascript-tracker/general-parameters/initializing-a-tracker-2/#Anonymous_Tracking_2150). 
+
+You may wish to toggle this functionality on or off during a page visit, for example when a user accepts a cookie banner you may not want to disable anonymous tracking, or when a user logs in to your site. Information on how to instrument this can be found in the [technical documentation](https://docs.snowplowanalytics.com/docs/collecting-data/collecting-from-own-applications/javascript-tracker/general-parameters/other-parameters-2/#Toggling_Anonymous_Tracking_2150).
```

Comment:
```suggestion
You may wish to toggle this functionality on or off during a page visit, for example when a user accepts a cookie banner you may want to disable anonymous tracking, or when a user logs in to your site. Information on how to instrument this can be found in the [technical documentation](https://docs.snowplowanalytics.com/docs/collecting-data/collecting-from-own-applications/javascript-tracker/general-parameters/other-parameters-2/#Toggling_Anonymous_Tracking_2150).
```